cse/src/main/java/com/adyen/checkout/cse/CardEncrypter.java

```diff
@@ -159,6 +176,9 @@ public static String encrypt(
             cardJson.put(EXPIRY_YEAR_KEY, unencryptedCard.getExpiryYear());
             cardJson.put(CVC_KEY, unencryptedCard.getCvc());
             cardJson.put(HOLDER_NAME_KEY, unencryptedCard.getCardHolderName());
+            cardJson.put(PASSWORD_KEY, unencryptedCard.getCardPassword());
+            cardJson.put(TAX_NUMBER_KEY, unencryptedCard.getTaxNumber());
+            // TODO ask caio why this is necessary
```

Comment:
@caiofaustino me and @jreij talked about this yesterday but couldn't figure out why this method is necessary. Can you explain why do we have this method?

Also we have an `UnencryptedCard` class used in this method with all the card fields and `EncryptedCard` with only the encrypted ones. However in this case I needed to add a KCP specific fields to `EncryptedCard` which only had fields that all cards need to have before. We only use fields from `UnencryptedCard` in `CardEncrypter`. Do you think it's necessary to have two different classes or should we have one class with all the fields where some of them are encrypted and some of them are not?

Reply:
Hmm this is indeed a new logic we didn't need until now, let me check with the other teams how they approach this.

Reply:
Keep in mind that if we leave these 2 fields here, we need to also add social security number.